authdelegate/Dockerfile

```diff
@@ -3,12 +3,9 @@ FROM dev-standard
 RUN bash -l -c 'go get github.com/18F/authdelegate && \
     strip $GOPATH/bin/authdelegate'
 
-RUN groupadd -r authdelegate && useradd -r -m -g authdelegate authdelegate && \
-    mkdir $APP_SYS_ROOT/authdelegate && \
-    chown authdelegate:authdelegate $APP_SYS_ROOT/authdelegate
```

Comment:
Yeah, sorry about that. This being my first foray into Docker, I'm making and rectifying rookie mistakes pretty quickly. What I realized I could do is use the `docker run` command with the `-v` option to mount the `config` directories, rather than explicitly make directories and `COPY` the configs into the image as I was doing previously here.  Since these servers only read configs, and don't need to write anything, there's no need to make any directories.